app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+# Post model
+class Post < ApplicationRecord
+  has_many :comments
+  belongs_to :user
```

Comment:
- Hi, according to the ERD diagram, **Post has many likes**. I would suggest adding it.